README.md

```diff
@@ -1,6 +1,106 @@
+# Logstash Filter Aggregate Documentation
```

Comment:
Yes, I think it's a good idea.
Which name do you prefer ? `BUILD.md` ? `DEVELOPER.md` ? Another ?